packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts

```diff
@@ -18,11 +18,11 @@ import {
   SfdxWorkspaceChecker
 } from '../util';
 
-import { getRootWorkspacePath } from '../../util';
+import { workspaceUtils } from '../../util';
 
 import { ConfigFile } from '@salesforce/core';
-import { isNullOrUndefined } from '@salesforce/salesforcedx-utils-vscode/out/src/helpers';
-import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import { isNullOrUndefined } from '@salesforce/salesforcedx-utils-vscode';
```

Comment:
`isNullOrUndefined` seems generic and not vs-specific.  Can it be moved from salesforcedx-utils-vscode to salesforcedx-utils?

Reply:
maybe address in a follow up ticket?

Reply:
100% agree. Will add it to my list of follow ups